pkg/config/config.go

```diff
@@ -94,6 +94,11 @@ const (
 	maxExternalMetricsProviderChunkSize = 35
 )
 
+// GetAllPossibleActivityDumpTracedEvents defines all the possible values for activity dumps traced events
+func GetAllPossibleActivityDumpTracedEvents() []string {
+	return []string{"exec", "open", "dns", "bind"}
+}
+
 // Datadog is the global configuration object
```

Comment:
Why not a const?

Reply:
Hi, that was my first attempt, but it appears that [go doesn't support slices constants](https://go.dev/ref/spec#Constants). If you think about a more elegant way, could you please share it ? :)